<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmMediaLIbrary</class>
 <widget class="QMainWindow" name="frmMediaLIbrary">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>722</width>
    <height>488</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>材料管理</string>
  </property>
  <property name="iconSize">
   <size>
    <width>18</width>
    <height>18</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupMediaList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>291</width>
      <height>471</height>
     </rect>
    </property>
    <property name="title">
     <string>材料库</string>
    </property>
    <widget class="QTableWidget" name="tbLibrary">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>271</width>
       <height>311</height>
      </rect>
     </property>
     <row>
      <property name="text">
       <string>1</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>2</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>3</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>类型</string>
      </property>
      <property name="textAlignment">
       <set>AlignJustify|AlignVCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>名称</string>
      </property>
      <property name="textAlignment">
       <set>AlignJustify|AlignVCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>所有者</string>
      </property>
      <property name="textAlignment">
       <set>AlignJustify|AlignVCenter</set>
      </property>
     </column>
     <item row="0" column="0">
      <property name="text">
       <string>Delectric</string>
      </property>
     </item>
     <item row="0" column="1">
      <property name="text">
       <string>D1</string>
      </property>
     </item>
     <item row="0" column="2">
      <property name="text">
       <string>System</string>
      </property>
     </item>
     <item row="1" column="0">
      <property name="text">
       <string>Delectric</string>
      </property>
     </item>
     <item row="1" column="1">
      <property name="text">
       <string>D2</string>
      </property>
     </item>
     <item row="1" column="2">
      <property name="text">
       <string>System</string>
      </property>
     </item>
     <item row="2" column="0">
      <property name="text">
       <string>Metal</string>
      </property>
     </item>
     <item row="2" column="1">
      <property name="text">
       <string>M1</string>
      </property>
     </item>
     <item row="2" column="2">
      <property name="text">
       <string>System</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="btnDeleteMedia">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>430</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>删除</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>81</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>筛选</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtFilter">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>161</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QToolButton" name="btnNew">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>430</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rdbSimple">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>简单模式</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rdbAdvanced">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>70</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>属性编辑</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnOK">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>430</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupModel">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>401</width>
      <height>471</height>
     </rect>
    </property>
    <property name="title">
     <string>介质材料</string>
    </property>
    <widget class="QLineEdit" name="txtMediaName">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>251</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>91</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>材料名称</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnAddMedia">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>430</y>
       <width>361</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>添加到材料库</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupFreq">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>381</width>
       <height>341</height>
      </rect>
     </property>
     <property name="title">
      <string>属性</string>
     </property>
     <widget class="QPushButton" name="btnAddFreq">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>300</y>
        <width>121</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>增加频点</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnRemoveFreq">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>300</y>
        <width>131</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>移除频点</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tbFreqList">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>361</width>
        <height>261</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
